addons/base_import/tests/test_cases.py

```diff
@@ -23,6 +26,12 @@ def sorted_fields(fields):
     recursed = [dict(field, fields=sorted_fields(field['fields'])) for field in fields]
     return sorted(recursed, key=lambda field: field['id'])
 
+def test_module_installed(module_name):
```

Comment:
Name doesn't really fit, this tests that the module is *not* installed (it returns `None` if the module is installed).